workflows/funcscan.nf

```diff
@@ -136,9 +136,11 @@ workflow FUNCSCAN {
             ch_annotation_faa        = PRODIGAL_GFF.out.amino_acid_fasta
             ch_annotation_fna        = PRODIGAL_GFF.out.nucleotide_fasta
             ch_annotation_gff        = PRODIGAL_GFF.out.gene_annotations
-            PRODIGAL_GBK ( ch_prepped_input, "gbk" )
-            ch_versions              = ch_versions.mix(PRODIGAL_GBK.out.versions)
-            ch_annotation_gbk        = PRODIGAL_GBK.out.gene_annotations
+            if ( params.save_annotations == true ) {
```

Comment:
Are you sure this is in the right place? What if you want to run PRODIGAL? but don't care about the annotations?

Or is this just for parity with PROKKA? :thinking: I can't remember now... sorry if I've got confused...

Reply:
The PRODIGAL_GBK is only needed to be run, if the user wishes to save the annotation in the output folder (to use it afterwards for secondary analysis). It is not needed in funcscan processes.